be/src/exec/pipeline/aggregate/spillable_aggregate_blocking_sink_operator.h

```diff
@@ -49,11 +49,10 @@ class SpillableAggregateBlockingSinkOperator : public AggregateBlockingSinkOpera
 
     size_t estimated_memory_reserved(const ChunkPtr& chunk) override {
         if (chunk && !chunk->is_empty()) {
-            if (_aggregator->is_hash_set()) {
```

Comment:
should we consider hash_set too?

Reply:
spillable aggregate block sink operator only use hash_map
spillable aggregate block distinct sink operator only use hash_set